purchases/src/main/kotlin/com/revenuecat/purchases/models/Transaction.kt

```diff
@@ -0,0 +1,20 @@
+package com.revenuecat.purchases.models
+
+import android.os.Parcelable
+import com.revenuecat.purchases.getDate
+import kotlinx.android.parcel.Parcelize
+import org.json.JSONObject
+import java.util.Date
+
+@Parcelize
+data class Transaction(
```

Comment:
Any better suggestion for this name?

Reply:
There's a `PurchaseHistoryRecord` in the BillingClient. I think we can go with `Transaction`